clients/js/src/ChromaClient.ts

```diff
@@ -94,7 +95,9 @@ export class ChromaClient {
   }
 
   /** @ignore */
-  init(): Promise<void> {
+  async init(): Promise<void> {
+    await this.resolveTenantAndDatabases();
```

Comment:
This is not optimal, as it causes us to make a call to the API every time. However, we can't call async code from the constructor so the alternative would be causing a breaking change for anyone using our JS client by forcing them to call a follow-on method upon initialization that sets the tenant/database. I propose we go about this way for now and implement the more efficient method when we apply more sweeping breaking client changes.